packages/artillery-plugin-apdex/index.js

```diff
@@ -66,6 +66,13 @@ class ApdexPlugin {
           return;
         }
 
+        if (testInfo.earlyStop) {
```

Comment:
I don't think we want to change current behavior. There is no clear reason to **not** always run these checks, so I'm hesitant to change existing behavior.

Reply:
okay - reverted. we'll address making cancellation more obvious at another point